data/en/license_faq.yaml

```
@@ -0,0 +1,20 @@
+- Question: Is Cryptomator free to use?
+  Answer: Cryptomator is free to use on Linux, macOS and Windows, but not on iOS and Android. If you purchase a license key for one platform, it is not valid for another. A bundle license does not exist.
+
+- Question: Why is Cryptomator free to use on some platforms, but not on others?
+  Answer: Our licensing strategy differs across platforms because of constraints imposed by Apple App Store policies, Google Play Store policies, and technical differences across desktop operating systems.
```

```suggestion
  Answer: Our licensing strategy differs across platforms because of constraints imposed by App Store / Google Play policies, and technical differences across desktop operating systems.
```

Just shortened a bit and used the up-to-date brand names.